docs/en/guides/How-to-release.md

```diff
@@ -103,18 +107,25 @@ This script takes care of the following things:
 1. Exclude all unnecessary files in the target source tar, such as `.git`, `.github`, and `.gitmodules`. See the script for more details.
 1. Execute `gpg` and `shasum 512`. 
 
-
 `apache-skywalking-apm-x.y.z-src.tgz` and files ending with `.asc` and `.sha512` may be found in the `tools/releasing` folder.
 
 ## Start the next iteration
 
 Once the release is deployed to the staging repositories, you can start updating the version to the next number and open a pull request.
 
 ```bash
-git checkout next
-./mvnw versions:set-property -DgenerateBackupPoms=false -Dproperty=revision -DnewVersion=<next-version-number>-SNAPSHOT
+# Update the version to the next snapshot version still in the same branch, such as 10.1.0-release
```

Comment:
There seems to make no sense to emphasize "still in the same branch`? Because we always use squash and merge and the tag commit won’t be available in our master branch anyway

Reply:
This is on the release branch. I want to several commits(1. change to release version 2. change to next snapshot, 3. update changelog) in one branch, and through one PR.

Reply:
OK, I thought you want to include the release commit in the master branch, which is impossible. Although 1 will be overridden by 2 finally, it doesn’t matter if they are in the same branch. LGTM

Reply:
On my side, I use the manual setup in the IDE, as there are too many changes for change logs. I usually don't do this in this way.